types/jest-matchers/index.d.ts

```diff
@@ -0,0 +1,2 @@
+import '../../packages/jest/src/index';
```

Comment:
Is this to avoid building? Or does `@compiled/jest` not work?

Reply:
Ignore for now still tweaking!